subjects/road_intersection/README.md

```diff
@@ -65,15 +65,17 @@ You are free to decide what algorithm you want to implement for the traffic ligh
 
 Vehicles must obey this rules:
 
-- Vehicles must have a color depending on their route the colors are up to you to decide(ex:`r`- purple, `s`- Blue and `l`- Yellow). This must then be given during the audit
+- Vehicles must have a color depending on their route the colors are up to you to decide 
 
-- Autonomous, vehicles driving on a lane with a **given route** must follow the direction of
-  that route, its not possible for the driver to change lanes or route.
+- Vehicles must have a color so that it's possible to identify which route it will follow (ex:`r`- purple, `s`- Blue and `l`- Yellow). This information about the colors must be given to the auditor of the raid.
+
+- Autonomous vehicles driving on a lane with a **given route** must follow the direction of
+that route, it's not possible for the driver to change lanes or routes.
```

Comment:
Autonomous vehicles don't have drivers.
Probable we can say something like:
> it is not possible for the autonomous vehicle to change lanes or routes.

Reply:
done